api/room/schema.py

```diff
@@ -189,7 +189,8 @@ class Arguments:
     @Auth.user_roles('Admin')
     def mutate(self, info, room_id, **kwargs):
         validate_empty_fields(**kwargs)
-        validate_structure_id(**kwargs)
+        if kwargs.get('structure_id'):
```

Comment:
If you validate for empty fields before checking for this, does it mean that you cannot update without structure id in the kwargs?

Reply:
When one provides an empty string you mean, am looking into that.